src/components/ContentWindow.jsx

```diff
@@ -0,0 +1,27 @@
+import { Route, Routes } from "react-router-dom"
+import ArticleList from "./ArticleList"
+import ArticlesPage from "./ArticlePage"
+import ArticlePage from "./ArticlePage"
+
+
+export default function ContentWindow() {
+    return (
+
+        <div id="content_window_container">
+            <p>ContentWindow</p>
```

Comment:
I don't think you need this here?

Reply:
Hi Mezz, this was just there so I could initially see it on the page to see that it's loading in when I create something new, it's more of a bug checking thing, will remove it when I have finished implementing everything and move onto styling